packages/generators/app/src/create-project.ts

```diff
@@ -98,6 +99,13 @@ export default async function createProject(
     // ensure node_modules is created
     await fse.ensureDir(join(rootPath, 'node_modules'));
 
+    // TODO: we only want to do that if we create a test app,
+    // to make sure yarn install works even when the folder is
+    // within a yarn workspace monorepo
+    if (scope.useYarn /* && isTestApp */) {
+      await fse.ensureFile(join(rootPath, 'yarn.lock'));
+    }
```

Comment:
@innerdvations Do you have an idea if I can add a flag in `Scope` to indicate if this is a test-app? I don't really like it, but don't have any better idea.